relayer/relayer/src/main.rs

```diff
@@ -214,6 +228,9 @@ async fn main() -> Result<(), RelayerError> {
                     tick = Instant::now();
                 }
             }
+            Err(RelayerError::TraderComponent(why)) => {
+                error!("Trader component exited with an error {why:?}. Restart is required to run Trader again.");
```

Comment:
Question: so if the trader crashes we ignore it and it won't be restarted? Doesn't this mean that any connection failure in the trader will just turn it on "forever" (until relayer restart)?